qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -860,6 +860,7 @@ def run(
             inputs: Program input parameters. These input values are passed
                 to the runtime program.
             options: Runtime options that control the execution environment.
+                ``job_tags`` and ``max_execution_time`` can also be set here.
```

Comment:
I think your intent is to move `job_tags` and `max_execution_time` to be part of `RuntimeOptions` class. If that's the case, then 1) the use of them as separate input args should be deprecated and 2) docstring should list them as bullet points, similar to `backend`, `image`, and `log_level`.

Reply:
Is this something we want to do? Deprecate `job_tags` and `max_execution_time` as input args and have them passed in through in `options` instead?

Reply:
Yeah since I think it's rather confusing to have some options inside `options` and some outside.